qemu-system-wrapper.sh

```diff
@@ -199,4 +203,5 @@ QEMU_ARGS=(
   -nodefaults
 )
 
+echo "QEMU cmd: \"$QEMU_X86_64\" \"${QEMU_ARGS[@]}\""
```

Comment:
[Argument mixes string and array. Use * or separate argument.](https://github.com/koalaman/shellcheck/wiki/SC2145)